src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -261,6 +261,38 @@ internal static CompletionResult GetCommandNameCompletionResult(string name, obj
             return new CompletionResult(name, listItem, CompletionResultType.Command, syntax);
         }
 
+        private static bool isNested(PSModuleInfo rootModule, PSModuleInfo childModule)
+        {
+            var nestedModules = rootModule.NestedModules;
```

Comment:
this line is unnecessary. It is not reduce any compute time nor ease the code readiness.